lib/src/components/NxPolicyViolationIndicator/NxPolicyViolationIndicator.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { forwardRef } from 'react';
+import { faExclamationCircle } from '@fortawesome/free-solid-svg-icons';
+
+import { ThreatLevelCategory, categoryByPolicyThreatLevel } from '../../util/threatLevels';
+import NxFontAwesomeIcon from '../NxFontAwesomeIcon/NxFontAwesomeIcon';
+
+import { Props, propTypes } from './types';
+export { Props } from './types';
+
+import './NxPolicyViolationIndicator.scss';
+
+const NxPolicyViolationIndicator = forwardRef<HTMLDivElement, Props>(
+  function NxPolicyViolationIndicator({ threatLevelCategory, policyThreatLevel, children }, ref ) {
+    const category: ThreatLevelCategory =
+          threatLevelCategory ? threatLevelCategory :
+          policyThreatLevel != null ? categoryByPolicyThreatLevel[policyThreatLevel] : 
+          'unspecified',
+
+        className = `nx-policy-violation-indicator nx-policy-violation-indicator--${category}`,
```

Comment:
Should allow custom classnames as well

Reply:
> Should allow custom classnames as well

https://github.com/sonatype/sonatype-react-shared-components/pull/229/commits/cc14aa0f2408a3148e653be8c45c748163c82150